update

```diff
@@ -1,10 +1,126 @@
+      if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]; then
+          # If the user agrees with it.
+          echo "%sExcellent! ${red}Please wait.${end}\n"
+          cd ../ && rm -rf ./$dir_name && git clone --recursive https://github.com/AimTuxOfficial/AimTux $dir_name && bash ./$dir_name/update
+      else
+          read -r -p "Please give the full path like shown above: " dir
+          cd ../ && rm -rf ./$dir && git clone --recursive https://github.com/AimTuxOfficial/AimTux $dir && bash ./$dir/update
+      fi
+  else
+      # Seen the user does not agree we are going to exit.
+      exit 1
+  fi
+}
+getMasterFree(){
+  read -r -p "Are you sure you would like to continue? All edits will be lost [y/N] " response
+  # Grab the response from the user input.
+  if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]; then
+      # If the user agrees with it.
+      echo "%sExcellent! ${red}Please do not exit this script until it is complete.${end}\n"
+      read -r -p "We are deleteing $dir_name, is this the correct directory? [y/N] " response
+      # Grab the response from the user input.
+      if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]; then
+          # If the user agrees with it.
+          echo "%sExcellent! ${red}Please wait.${end}\n"
+          cd ../ && rm -rf ./$dir_name && git clone -b master-freetype https://github.com/AimTuxOfficial/AimTux $dir_name && bash ./$dir_name/update
+      else
+          read -r -p "Please give the full path like shown above: " dir
+          cd ../ && rm -rf ./$dir && git clone -b master-freetype https://github.com/AimTuxOfficial/AimTux $dir && bash ./$dir/update
+      fi
+  else
+      # Seen the user does not agree we are going to exit.
+      exit 1
+  fi
+}
+
+getFace(){
+  read -r -p "Are you sure you would like to continue? All edits will be lost [y/N] " response
+  # Grab the response from the user input.
+  if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]; then
+      # If the user agrees with it.
+      echo "%sExcellent! ${red}Please do not exit this script until it is complete.${end}\n"
+      read -r -p "We are deleteing $dir_name, is this the correct directory? [y/N] " response
+      # Grab the response from the user input.
+      if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]; then
+          # If the user agrees with it.
+          echo "%sExcellent! ${red}Please wait.${end}\n"
+          cd ../ && rm -rf ./$dir_name && git clone -b faceit https://github.com/AimTuxOfficial/AimTux $dir_name && bash ./$dir_name/update
+      else
+          read -r -p "Please give the full path like shown above: " dir
+          cd ../ && rm -rf ./$dir && git clone -b faceit https://github.com/AimTuxOfficial/AimTux $dir && bash ./$dir/update
+      fi
+  else
+      # Seen the user does not agree we are going to exit.
+      exit 1
+  fi
+}
+
+_showMenu(){
+
+echo "<========================================>"
+echo "         Please select branch"
+echo "<========================================>"
+echo "[1] Master"
```

Comment:
"if users downloaded a branch other than the master branch"

By the menu, it allows them to choose what they want.